test/server/listener_manager_impl_test.cc

```diff
@@ -486,26 +486,6 @@ TEST_F(ListenerManagerImplWithRealFiltersTest, TerminalNotLast) {
       "filter in a network filter chain.");
 }
 
-TEST_F(ListenerManagerImplWithRealFiltersTest, NotTerminalLast) {
-  const std::string yaml = R"EOF(
```

Comment:
Can you revert these changes if they make no functional change? Just to better preserve git history

Reply:
fixed.